sites/next.skeleton.dev/src/components/docs/ThemeSwitch.astro

```diff
@@ -29,15 +28,15 @@ import { Palette } from 'lucide-react';
 	// Change Handler
 	function onChangeHandler(event) {
 		const selectedTheme = event.target.value;
-		elemBody.dataset.theme = selectedTheme;
+		document.body.dataset.theme = selectedTheme;
```

Comment:
I have no qualms with this change, just curious what the benefit to this approach was over the cached element query approach used previously.

Reply:
Querying the body is obsolete, the `body` reference is always available through `document.body` so we were querying what we already had.